release-1.9/release-1.9.md

```diff
@@ -0,0 +1,89 @@
+# Proposed timeline for 1.9
+We will begin the 1.9 release the first week of October 2017.
+
+The proposal below is similar in layout previous releases:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+- there are a number of North American holidays during Q4
+- same as in 1.8
+  * alpha's should not be cut unless all master-release-blocking tests are passing
```

Comment:
In practice this didn't work, because we never got the tests passing.  Do anticipate things being better for 1.9?